vlib/v/checker/tests/assign_const_ptr_int_literal_err.out

```diff
@@ -0,0 +1,14 @@
+vlib/v/checker/tests/assign_const_ptr_int_literal_err.vv:4:2: warning: unused variable: `b`
+    2 |
+    3 | fn main() {
+    4 |     b := &a
+      |     ^
+    5 | }
+Details: Specify the type for the constant value. Example:
```

Comment:
Seems like these details should come _after_ the error message.

Reply:
I thought so too at first but checking other tests, the Details came above.

E.g. running examples taken from other tests:
```
// vlib/v/checker/tests/array_sort_with_compare_err.vv
fn main() {
	mut names := ['aaa', 'bbb', 'ccc']

	names.sort_with_compare(sort_by_file_base)
	println(names)

	names.sort_with_compare(22)
	println(names)
}

fn sort_by_file_base(a string, b string) int {
	return int(a > b)
}
```

```v
// vlib/v/checker/tests/unimplemented_interface_e.vv
interface Animal {
	speak(s string)
}

struct Cat {}

fn (c Cat) speak(s &string) {}

fn foo(a Animal) {}

fn main() {
	foo(Cat{})
	_ = Animal(Cat{})
}
```

Reply:
Though I should not make the added tests produce an unused error